lib/roster.ts

```diff
@@ -0,0 +1,46 @@
+import { AscensionLevel, CharacterClass, Faction } from "./characters";
+
+export enum Category {
+  Starter = "starter",
+  Seasonal = "seasonal",
```

Comment:
we spoke about having the ability to support multiple different seasons. i don't see that reflected in this design. what are your plans for that?

Reply:
So in terms of the roster editor, I feel as though we should only show the active season and allow that to be edited. Come the new season, you won't be able to look back at previous seasons. On the profile page and in the json that holds all the data, those will have an active flag for a season. If the season is inactive, you won't be able to modify it in the roster editor. However on the profile page, we can show all the data the user has and seasons included with a dropdown.